src/spox/_public.py

```diff
@@ -37,6 +40,32 @@ def argument(typ: Type) -> Var:
     ).outputs.arg
 
 
+def initializer(value: npt.ArrayLike, dtype: npt.DTypeLike = None) -> Var:
+    """
+    Create a Var with a constant value.
+
+    Parameters
+    ----------
+    value
+        Array-like value for the variable.
+    dtype
+        Data type for the given value. If ``None``, it is inferred from the value
+        using numpy rules (``numpy.array(value)``).
+
+    Returns
+    -------
+    Var
+        Variable with the given constant ``value``.
+
+    Notes
+    -----
+    When the model is built, constants created by this function become initializers.
```

Comment:
Could you add a brief comment about how this initializer differs from constant nodes in ONNX (or provide a link to the ONNX IR)?